src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -1,27 +1,27 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-using System;
-using System.Collections.Generic;
+﻿// Copyright (c) Microsoft Corporation.

```

Comment:
Why are there so many diffs in this file? Line ending change?

Reply:
Dont know to be honest. From CodeFlow, everything looks good.